functions/wordpress/gravityForms/processGfFormSubmission.ts

```diff
@@ -0,0 +1,30 @@
+// Internal dependencies.
+import mutationSubmitForm from "@/lib/wordpress/gravityForms/mutationSubmit"
+import { formatGFSubmissionData } from "./formatGFSubmissionData"
+
+export default async function processGfFormSubmission(
+  formId: number | string,
+  values: any,
```

Comment:
@Danishkhurshid Please keep in mind `any` is never an option. If you need to add it do to time, please add a @TODO comment so it can be addressed later.